eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -105,6 +123,27 @@ function List-PullRequests {
   return Invoke-GitHubAPIGet -apiURI $uri
 }
 
+# 
+<#
+.PARAMETER Ref
+Ref to search for
+Pass 'heads/<branchame> ,tags/<tag name>, or nothing
+#>
+function List-References {
+  param (
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+    $Ref
```

Comment:
At some point we might also need to pass a token here if we ever call this on a private repo for example.